### PR TITLE
[UPDATE]: update version of mongoose mocha chai + fixing vulnerabilities

### PR DESCRIPTION
**Pull request by ecanavesio**

### CODE REVIEW

**Comment by Nicodav28** [comment]
@dsanel  Could you please merge this? It would be a great update and without a doubt those of us who use your library would be immensely grateful. pls :D

**Comment by dsanel** [comment]
@Nicodav28 @ecanavesio I will check this and several other PR and will create a new version with several updates. Thx for help

File: index.js [hunk 1]
```diff
@@ -289,28 +287,18 @@ module.exports = function (schema, options) {
...
+        return this.deleteMany(conditions, deletedBy);
     };
 
-    schema.statics.restore =  function (conditions, callback) {
-        if (typeof conditions === 'function') {
-            callback = conditions;
-            conditions = {};
-        }
-
+    schema.statics.restoreMany =  function (conditions) {
```
Comment (dsanel) [inline index.js:293]: @ecanavesio Any specific reason why you replaced the `restore` method with `restoreMany`?

Comment (ecanavesio) [reply index.js:293]: The last version of mongoose doesn't have `restore` method

**Pull request closed without merging**